inc/cachify.class.php

```diff
@@ -1124,6 +1086,80 @@ private static function _is_logged_in() {
 		return false;
 	}
 
+	/**
+	 * Register all hooks to flush the total cache
+	 *
+	 * @since   2.4.0  Make the function public
+	 *
+	 * @param   void
+	 * @return  void
+	 */
+	private static function register_flush_cache_hooks(){
+
+		/* Define all default flush cache hooks */
+		$flush_cache_hooks = array(
+			array(
+				'action' => 'cachify_flush_cache',
+				'priority' => 10
+			),
+			array(
+				'action' => '_core_updated_successfully',
+				'priority' => 10
+			),
+			array(
+				'action' => 'switch_theme',
+				'priority' => 10
+			),
+			array(
+				'action' => 'before_delete_post',
+				'priority' => 10
+			),
+			array(
+				'action' => 'wp_trash_post',
+				'priority' => 10
+			),
+			array(
+				'action' => 'create_term',
+				'priority' => 10
+			),
+			array(
+				'action' => 'delete_term',
+				'priority' => 10
+			),
+			array(
+				'action' => 'edit_terms',
+				'priority' => 10
+			),
+			array(
+				'action' => 'user_register',
+				'priority' => 10
+			),
+			array(
+				'action' => 'edit_user_profile_update',
+				'priority' => 10
+			),
+			array(
+				'action' => 'delete_user',
+				'priority' => 10
+			)
+		);
+
+		$flush_cache_hooks = apply_filters( 'cachify_flush_cache_hooks', $flush_cache_hooks );
+
+		/* Loop all hooks and register actions */
+		foreach ($flush_cache_hooks as $hook) {
+			add_action(
```

Comment:
Please, add 4-th parameter (`$accepted_args`) to [add_action](https://developer.wordpress.org/reference/functions/add_action/) and set it to `0`. `Cachify::flush_total_cache()` accepts one parameter (`$clear_all_methods`), but I'm sure you don't want to have it set in this situation and it is unrelated to whatever the hooked actions pass as first parameter (like `$term_id` in case of [create_term](https://developer.wordpress.org/reference/hooks/create_term/) action etc.)

Reply:
changed in f97b8157985ed0ff1125dac6cfba40a6bfd436e0